test/meson.build

```diff
@@ -540,6 +540,7 @@ foreach target : targets
                       'test-ungetc-ftell',
                       'test-fgetc',
                       'test-fgets-eof',
+                      'test-wchar',
```

Comment:
It would be good to run this test on the native library to make sure picolibc and glibc (the usual native library) agree on these semantics.

Reply:
```
test('test-wchar',
         executable('test-wchar',
                    'test-wchar.c',
                    c_args: native_c_args,
                    link_args: native_c_args,
                    dependencies: native_lib_m))
```
@keith-packard, I'm not 100% sure about my understanding of the meson.build file, do you mean i need to add the test like that at line 625 instead of what i did ?

Reply:
Yes, but you need to add the native testing in addition to the regular test -- that way when doing "native" testing, we'll test the picolibc implementation and then also test the host C library implementation to validate the test itself. So, add what you've got above somewhere around line 625, yes, but also add `'test-wchar',` to the list of simple tests.